src/makielayout/blocks/legend.jl

```diff
@@ -450,6 +519,9 @@ _renaming_mapping(::Type{PolyElement}) = Dict(
     :colormap => :polycolormap,
     :colorrange => :polycolorrange,
 )
+_renaming_mapping(::Type{MeshElement}) = Dict()
+_renaming_mapping(::Type{ImageElement}) = Dict()
+_renaming_mapping(::Type{MeshScatterElement}) = Dict()
```

Comment:
What's the point of these? They just seem to map the simpler attribute names in a LegendElement to more complicated ones like `color -> polycolor`. But the attributes are already isolated there. Am I missing something?

Reply:
@jkrumbiegel Is it fine to leave these?